### DISCUSSION TITLE
How can I add customfield when creating an issue?

### DISCUSSION THREAD

**Discussion by jbertozzi** [ankitpokhrel/jira-cli#289] [Q&A] [answered]
I'd like to know if this is possible. My company have made some (custom) fields mandatory and thus cannot I create issue using this tool.

**Comment by ankitpokhrel** [comment]
Hi @jbertozzi, this is not supported at the moment. There is already an open request for this: https://github.com/ankitpokhrel/jira-cli/discussions/222. I will work on it as soon as I have some time.

**Comment by ankitpokhrel** [comment, marked as answer]
This change is available in the new release [v1.0.0-beta](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.0.0-beta)
